### PR TITLE
#288

### PR DESCRIPTION
**Pull request by igaidai4uk**

### CODE REVIEW

File: server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java [hunk 1]
```diff
@@ -22,11 +24,14 @@
 public class WebSocketHandler extends SimpleChannelInboundHandler<Object> {
 
     public static final String WEBSOCKET_PATH = "/websocket";
+    protected final GlobalStats globalStats;
```
Comment (doom369) [inline server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java]: Why protected?

File: server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java [hunk 2]
```diff
@@ -56,6 +61,8 @@ public void channelRead0(ChannelHandlerContext ctx, Object msg) {
         } else if (msg instanceof WebSocketFrame) {
             handleWebSocketFrame(ctx, (WebSocketFrame) msg);
         }
+
+        globalStats.mark(Command.WEB_SOCKETS);
```
Comment (doom369) [inline server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java]: I think it is better to move counter to `handleWebSocketFrame` method for `BinaryWebSocketFrame`

Comment (igaidai4uk) [inline server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java]: Do you think? I thought that `channelRead0` would only get websocket message.

Comment (doom369) [inline server/web-socket/src/main/java/cc/blynk/server/websocket/handlers/WebSocketHandler.java]: channelRead0 is triggered every time message is read from socket. This is useful metric for server monitoring especially when I need to understand where is hot place.

**Pull request merged**